app/lib/advanced_filter_constants.rb

```diff
@@ -0,0 +1,472 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/ModuleLength
+
+# Options for Advanced Filter
+module AdvancedFilterConstants
```

Comment:
sorry I know I had originally suggested calling this module `AdvancedFilterConstants` to mirror `ImportExportConstants` but since these options are dynamic, can we rename it to `AdvancedFilterOptions`?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1239/commits/7057755586b708f2ac49b5a7c8d615a84a5781bf